lms/djangoapps/commerce/views.py

```diff
@@ -79,6 +79,23 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
         if not valid:
             return DetailResponse(error, status=HTTP_406_NOT_ACCEPTABLE)
 
+        # Ensure that the course has an honor mode with SKU
+        honor_mode = CourseMode.mode_for_course(course_key, CourseMode.HONOR)
```

Comment:
Disregard. I was reading the history in the wrong order. Also, other merged PRs seem to exhibit the same behavior.